packages/testlab/src/validate-api-spec.ts

```diff
@@ -3,20 +3,15 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import * as SwaggerParser from 'swagger-parser';
-import {OpenApiSpec} from '@loopback/openapi-spec';
+import {OpenApiSpec} from '@loopback/openapi-spec-types';
+const validator = require('swagger2openapi/validate.js');
+import * as util from 'util';
+const promisify = util.promisify;
+const promisifiedValidator = promisify(validator.validate);
```

Comment:
I find this naming convention rather unusual. Functions and methods should start with a verb.

I am proposing to use the name `promisifyAsync` instead.

Reply:
> I am proposing to use the name promisifyAsync instead.

Uh oh, I meant `validateAsync`.